src/styles/main.scss

```
@@ -70,6 +74,6 @@ body {
 
 @each $week-days, $n in $week-days {
   .calendar-#{$week-days} :first-child {
-    margin-left: $n * ($cell-size + 1px);
+    margin-left: $n * ($cell-size)+$margin-cells*3;
```

```suggestion
    margin-left: $n * ($cell-size) + $margin-cells * 3;
```
don't forget about readability

---

ok. thanks a lot